pyproject.toml

```diff
@@ -182,11 +186,13 @@ testpaths = [
 env = []
 
 
-# https://python-semantic-release.readthedocs.io/en/latest/
-[tool.semantic_release]
-branch = "main"
-version_variable = "src/package/__init__.py:__version__"
-upload_to_pypi = false
-commit_subject = "bump: release v{version}"
-changelog_file = "CHANGELOG.md"
-build_command = "pip install 'flit >=3.2.0,<4' && flit build"
+# https://github.com/commitizen-tools/commitizen
+# https://commitizen-tools.github.io/commitizen/bump/
+[tool.commitizen]
+bump_message = "bump: release $current_version → $new_version"
+tag_format = "v$major.$minor.$patch$prerelease"
+update_changelog_on_bump = true
+version_files = [
+    "src/package/__init__.py:__version__",
+]
+version = "2.1.0"
```

Comment:
I try to sort the entries in the `pyproject.toml` file alphabetically, so maybe we can move this one after

https://github.com/jenstroeger/python-package-template/blob/1cd7aa0cb52b2395aeef33c8f46b914ac45b92e4/pyproject.toml#L76-L78